### PR TITLE
fix: add Vee339 

### LINKED ISSUE #1685: [PROFILE ADDITION] <Veerpal Kaur>
### Profile screenshot

my_github_profile

### PR DESCRIPTION
**Pull request by Vee339**
resolves #1685

### CODE REVIEW

File: profiles/vee339.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/81091817/214922721-3a408d17-fc53-4278-b75e-a529663bcc97.png",
```
Comment (Panquesito7) [inline profiles/vee339.json:2]: I recommend more of an image like this. What do you think? 😃
P.S.: the badges seem to be failing due to an issue with the repository, I think (not being caused by the way you set it up).

image

**Comment by Vee339** [comment]
@Panquesito7 What do you mean? Please explain me.

**Comment by Panquesito7** [comment]
> @Panquesito7 What do you mean? Please explain me.

Regarding the issue of the badges or? 🤔

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻 ⚡

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**